### DISCUSSION TITLE
context deadline exceeded

### DISCUSSION THREAD

**Discussion by huangxueliang** [milvus-io/milvus#37645] [Show and tell]
{"log":"[2024/11/13 08:02:43.287 +00:00] [INFO] [hookutil/hook.go:46] [\"empty so path, skip to load plugin\"]\n","stream":"stdout","time":"2024-11-13T08:02:43.287871745Z"}
{"log":"[2024/11/13 08:02:43.287 +00:00] [DEBUG] [proxy/service.go:122] [\"create a new Proxy instance\"] [state=2]\n","stream":"stdout","time":"2024-11-13T08:02:43.288058526Z"}
{"log":"[2024/11/13 08:02:43.288 +00:00] [DEBUG] [proxy/service.go:423] [\"init Proxy server\"]\n","stream":"stdout","time":"2024-11-13T08:02:43.288249045Z"}
{"log":"[2024/11/13 08:02:43.288 +00:00] [DEBUG] [proxy/service.go:454] [\"Proxy init service's parameter table done\"]\n","stream":"stdout","time":"2024-11-13T08:02:43.288424282Z"}
{"log":"[2024/11/13 08:02:43.288 +00:00] [DEBUG] [proxy/service.go:456] [\"Proxy init http server's parameter table done\"]\n","stream":"stdout","time":"2024-11-13T08:02:43.288573452Z"}
{"log":"[2024/11/13 08:02:43.288 +00:00] [DEBUG] [proxy/service.go:463] [\"init Proxy's parameter table done\"] [internalAddress=172.26.0.4:19529] [externalAddress=172.26.0.4:19530]\n","stream":"stdout","time":"2024-11-13T08:02:43.288871483Z"}
{"log":"[2024/11/13 08:02:43.289 +00:00] [INFO] [accesslog/writer.go:98] [\"Access log save to /tmp/milvus\"]\n","stream":"stdout","time":"2024-11-13T08:02:43.289540505Z"}
{"log":"[2024/11/13 08:02:43.290 +00:00] [INFO] [accesslog/global.go:145] [\"Init access logger success\"]\n","stream":"stdout","time":"2024-11-13T08:02:43.290889937Z"}
{"log":"[2024/11/13 08:02:43.290 +00:00] [DEBUG] [proxy/service.go:470] [\"init Proxy's tracer done\"] [\"service name\"=\"Proxy ip: 172.26.0.4, port: 19530\"]\n","stream":"stdout","time":"2024-11-13T08:02:43.291086661Z"}
{"log":"[2024/11/13 08:02:43.291 +00:00] [INFO] [etcd/etcd_util.go:49] [\"create etcd client\"] [useEmbedEtcd=false] [useSSL=false] [endpoints=\"[etcd:2379]\"] [minVersion=1.3]\n","stream":"stdout","time":"2024-11-13T08:02:43.291325304Z"}
{"log":"[2024/11/13 08:02:43.290 +00:00] [INFO] [runtime/asm_amd64.s:1650] [\"Start check query node health loop\"]\n","stream":"stdout","time":"2024-11-13T08:02:43.291957928Z"}
{"log":"[2024/11/13 08:02:43.385 +00:00] [INFO] [etcd/etcd_util.go:49] [\"create etcd client\"] [useEmbedEtcd=false] [useSSL=false] [endpoints=\"[etcd:2379]\"] [minVersion=1.3]\n","stream":"stdout","time":"2024-11-13T08:02:43.385714092Z"}
{"log":"[2024/11/13 08:02:48.275 +00:00] [DEBUG] [rootcoord/service.go:198] [\"RootCoord connect to etcd failed\"] [error=\"context deadline exceeded\"]\n","stream":"stdout","time":"2024-11-13T08:02:48.275594822Z"}
{"log":"[2024/11/13 08:02:48.275 +00:00] [**ERROR**] [components/root_coord.go:55] [\"RootCoord starts error\"] [error=\"context deadline exceeded\"] [stack=\"github.com/milvus-io/milvus/cmd/components.(*RootCoord).Run\\n\\t/workspace/source/cmd/components/root_coord.go:55\\ngithub.com/milvus-io/milvus/cmd/roles.runComponent[...].func1\\n\\t/workspace/source/cmd/roles/roles.go:114\"]\n","stream":"stdout","time":"2024-11-13T08:02:48.27587644Z"}
{"log":"panic: context deadline exceeded\n","stream":"stderr","time":"2024-11-13T08:02:48.356982318Z"}
{"log":"\n","stream":"stderr","time":"2024-11-13T08:02:48.357020893Z"}

**Comment by huangxueliang** [comment]
how can i slove issue?

**Reply by yhmo** [reply]
"RootCoord connect to etcd failed" ------------------- etcd is not alive

Normally, there should be 3 containers: etcd, minio, milvus
![Screenshot from 2024-11-13 16-49-46](https://github.com/user-attachments/assets/a80a8394-f7c4-4a0a-95e6-00ca77c13288)

**Comment by huangxueliang** [comment]
this is Standalone env,but i start the milvus(docker-compose start standalone),it is ok,why?

**Reply by yhmo** [reply]
I suppose the command is "docker compose up -d", this command starts 3 containers. Use command "docker ps" to observer how many containers are running. You should learn more about docker and docker compose.
